doc/nrf/config_and_build/bootloaders_and_dfu/bootloader_adding.rst

```diff
@@ -286,35 +297,39 @@ The following section describes how to add it to your secure bootloader chain.
 Adding MCUboot as an upgradable bootloader
 ==========================================
 
-To use MCUboot as an upgradable bootloader, the application must already use the |NSIB| as the immutable bootloader.
-MCUboot is added to its boot chain by including the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option with either the build command or in the application's :file:`prj.conf` file:
+To use MCUboot as an upgradable bootloader to your application, complete the following steps:
 
-.. code-block::
+1. :ref:`Add |NSIB| as the immutable bootloader <ug_bootloader_adding_immutable_b0>`.
+#. Add MCUboot to the boot chain by including the :kconfig:option:`CONFIG_BOOTLOADER_MCUBOOT` Kconfig option with either the build command or in the application's :file:`prj.conf` file:
 
-   west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
-   -DCONFIG_SECURE_BOOT=y \
-   -DCONFIG_BOOTLOADER_MCUBOOT=y
+   .. code-block::
 
-|how_to_configure|
+      west build -b nrf52840dk_nrf52840 zephyr/samples/hello_world -- \
+      -DCONFIG_SECURE_BOOT=y \
+      -DCONFIG_BOOTLOADER_MCUBOOT=y
 
-MCUboot can use the cryptographic functionality exposed by the immutable bootloader, reducing the flash memory usage for MCUboot to less than 16 kB.
-To enable this configuration, apply both the :file:`prj_minimal.conf` Kconfig project file and the :file:`external_crypto.conf` Kconfig fragment for the MCUboot image:
+   |how_to_configure|
 
-.. code-block::
+#. Optionally, you can configure MCUboot to use the cryptographic functionality exposed by the immutable bootloader and reduce the flash memory usage for MCUboot to less than 16 kB.
+   To enable this configuration, apply both the :file:`prj_minimal.conf` Kconfig project file and the :file:`external_crypto.conf` Kconfig fragment for the MCUboot image:
```

Comment:
Maybe we should mention that users in this case can use [CONFIG_PM_PARTITION_SIZE_MCUBOOT](https://github.com/nrfconnect/sdk-nrf/blob/d5eb58b3ce993e460e18b0119a19a4e7b70ec126/modules/mcuboot/boot/zephyr/Kconfig#L21) on the child image to redcue the partition size to match?

Reply:
And yes, this configuration is not to find in Kconfig search, as MCUboot is an "application" and not part of the Kconfig system as such. Have been discussed in tech-pubs teams before I think

Reply:
We should mention [CONFIG_PM_PARTITION_SIZE_MCUBOOT](https://github.com/nrfconnect/sdk-nrf/blob/d5eb58b3ce993e460e18b0119a19a4e7b70ec126/modules/mcuboot/boot/zephyr/Kconfig#L21) in the "Customizing the bootloader" section as well, as this configuration is not mentioned once in the docs. [Search tests](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/search.html?q=+CONFIG_PM_PARTITION_SIZE_MCUBOOT).
But that is maybe for another PR?

Reply:
The `CONFIG_PM_PARTITON_SIZE_MCUBOOT` is used only if we use dynamic partition map. Maybe we could describe this possibility in Partition Manager docs and link there.

Reply:
Agreed.
Partition Manager docs should definetly meantion CONFIG_PM_PARTITION_SIZE_<CHILD_IMAGE> and list the common variants of this

Reply:
https://nordicsemi.atlassian.net/browse/NCSDK-22733

Reply:
Attempt to address these points here: https://github.com/nrfconnect/sdk-nrf/pull/15018